sphinx6-preparations.rst

```diff
@@ -0,0 +1,37 @@
+.. post:: Dec 5, 2022
+   :tags: newsletter, python
+   :author: Ben
+   :location: MLM
+
+.. meta::
+   :description lang=en:
+      Sphinx 6 is coming soon. Here are our thoughts about preparations that we
+      are making, which may affect projects looking to upgrade.
+
+
+Sphinx 6 is coming
```

Comment:
This is a very tongue in cheek headline, and doesn't provide the user much value. We should give more details:

```suggestion
Sphinx 6 is coming soon and has important breaking changes
```